Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -0,0 +1,29 @@
+- type: entity
+  name: energy sword
+  parent: BaseItem
+  id: EnergySword
+  description: A very dangerous energy sword. Can be stored in pockets when turned off. Makes a lot of noise when used or turned on.
+  components:
+  - type: Esword
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/e_sword.rsi
+    layers:
+      - state: e_sword
+      - state: e_sword_blade
+        color: "#FFFFFF"
+        visible: false
+        shader: unshaded
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 7
+  - type: Item
+    size: 5
+    sprite: Objects/Weapons/Melee/e_sword.rsi
+  - type: UseDelay
+    delay: 1.0
+  - type: PointLight
+    enabled: false
+    radius: 1.5
```

Comment:
can probably be bumped up to
```yaml
radius: 2
energy: 2
```
its not darth-vader-hallway-scene-ey enough when using it in dark maints tunnels

Reply:
Thanks for reminding me! I'll prolly be adding lights in the next, follow-up PR.
I got some life stuff coming up tomorrow and I prolly won't be able to get back to this for a while, so I just wanted to finish this one PR for now, and I'll add lights and other sword variants later!

Reply:
Also now that there's a point light you should turn the light on/off (`SharedPointLightComponent.Enabled = ...`) and the colour (`SharedPointLightComponent.Color`)